chassis/invokers/src/main/java/com/griddynamics/jagger/invoker/http/v2/DefaultHttpInvoker.java

```diff
@@ -0,0 +1,28 @@
+package com.griddynamics.jagger.invoker.http.v2;
+
+import com.google.common.base.Preconditions;
+import com.griddynamics.jagger.invoker.InvocationException;
+
+import static java.lang.String.format;
+
+/**
+ * Created by aantonenko on 9/29/16.
+ */
+@SuppressWarnings("unused")
+public class DefaultHttpInvoker extends AbstractHttpInvoker {
+
+    public DefaultHttpInvoker(JHttpClient httpClient) {
```

Comment:
Could you please elucidate, what is the biggest difference comparing to the super class? This one looks almost the same. And as a user I still have to inject http client here. What if we use default client inside default invoker?